src/main/java/org/zalando/nakadi/domain/EventTypeOptions.java

```diff
@@ -1,10 +1,7 @@
 package org.zalando.nakadi.domain;
 
-import javax.annotation.Nullable;
-
 public class EventTypeOptions {
 
-    @Nullable
     private Long retentionTime;
```

Comment:
Like this https://github.com/zalando/nakadi/blob/master/src/main/java/org/zalando/nakadi/domain/EventType.java#L41